packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -563,12 +570,12 @@ class DebuggerController extends DisposableController
     // We're resuming and another isolate has started in a paused state,
     // resume any pauseState isolates.
     if (_resuming.value &&
-        event.isolate.id != isolateRef?.id &&
+        event.isolate!.id != isolateRef.id &&
```

Comment:
put `event.isolate!.id` into a local variable at the top of this method. This prevents one null-assert